Sections/EEA/GPPExtension: IAB Europe TCF.md

```diff
@@ -129,30 +135,48 @@ The core segment must always be present. It consists of the following fields:
+  <td>List of Vendor IDs that indicate Consent for these vendors. Please note that the field is composed of several fields in order to store the data in GPP string format. The client side API format is an array of int.<br><br>Equivalent of the fields listed under the Vendor Consent Section as defined in the <a href="https://github.com/InteractiveAdvertisingBureau/GDPR-Transparency-and-Consent-Framework/blob/master/TCFv2/IAB%20Tech%20Lab%20-%20Consent%20string%20and%20vendor%20list%20formats%20v2.md">TCF v2 specification:</a> <br>MaxVendorId, <br>IsRangeEncoding,<br>BitField, NumEntries, IsARange, StartOrOnlyVendorId, EndVendorId</td>
   </tr>
   <tr>
-  <td>VendorLegitimateInterest</td>
+  <td>VendorLegitimateInterest*</td>
   <td>OptimizedIntRange</td>
-  <td>List of Vendor IDs for which Legitimate Interest is established and the user did not exercise their “Right to Object”. Please note that the field is composed of several fields in order to store the data in GPP string format. The client side API format is array of int.<br></br>Equivalent of the fields listed under the Vendor Legitimate Interest Section as defined in the <a href="https://github.com/InteractiveAdvertisingBureau/GDPR-Transparency-and-Consent-Framework/blob/master/TCFv2/IAB%20Tech%20Lab%20-%20Consent%20string%20and%20vendor%20list%20formats%20v2.md">TCF v2 specification:</a><br></br>MaxVendorId, <br></br>IsRangeEncoding, <br></br>BitField, NumEntries, IsARange, StartOrOnlyVendorId, EndVendorId </td>
+  <td>List of Vendor IDs for which Legitimate Interest is established and the user did not exercise their “Right to Object”. Please note that the field is composed of several fields in order to store the data in GPP string format. The client side API format is array of int.<br><br>Equivalent of the fields listed under the Vendor Legitimate Interest Section as defined in the <a href="https://github.com/InteractiveAdvertisingBureau/GDPR-Transparency-and-Consent-Framework/blob/master/TCFv2/IAB%20Tech%20Lab%20-%20Consent%20string%20and%20vendor%20list%20formats%20v2.md">TCF v2 specification:</a><br>MaxVendorId,<br>IsRangeEncoding,<br>BitField, NumEntries, IsARange, StartOrOnlyVendorId, EndVendorId</td>
+  </tr>
+  <tr style="background-color:#000;color:#FFF;">
+    <td>
+      <strong>Publisher Restrictions Section</strong>
+    </td>
+    <td colspan=2>The content of this section is optional EXCEPT for NumPubRestrictions. Encodes any number of single or range restriction entries.
+    </td>
   </tr>
   <tr>
   <td>NumPubRestrictions</td>
   <td>Int(12)</td>
-  <td>Number of restriction records to follow.<br></br>Value is required even if it is <code>0</code></td>
+  <td>Number of restriction records to follow.<br>Value is required even if it is <code>0</code></td>
```

Comment:
needs to be removed and replaced with one field of type ArrayOfRanges (replaces NumPubRestrictions, PurposeID, RestrictionType, PubRestrictionEntry)

Reply:
Fixed, @janwinkler please take a look as the new section.

Reply:
@HeinzBaumann maybe just clarification on the last line:
"A single or range of Vendor ID(s) who the publisher has designated as restricted under the purpose id."
--> "A list of Vendor ID(s) who the publisher has designated as restricted under the purpose id encoded as bitmask or ranges."

Reply:
Thanks! I made this modification.